.github/workflows/smoke-test.yaml

```diff
@@ -0,0 +1,17 @@
+name: Action Smoke Tests
+on: workflow_dispatch
+jobs:
+  run-action:
+    name: Run GH Action
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        name: Checkout Repository
+      - uses: ./
+        name: Run GH Action on ${{ github.ref }}
+        env:
+          CSP_API_TOKEN: ${{ secrets.CSP_API_TOKEN }}
+          GITHUB_EVENT_PATH: .github/.vib/github-event.json
+          VIB_ENV_TARGET_PLATFORM: 91d398a2-25c4-4cda-8732-75a3cfc179a1
+        with:
+          config: .github/.vib
```

Comment:
Won't work. You have it on .github/workflows/.vib

Reply:
Good catch, fixed in 523d743